src/pfe/file-watcher/server/test/test.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+# Colors for success and error messages
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+BLUE='\033[0;36m'
+RESET='\033[0m'
+
+function usage {
+    me=$(basename $0)
+    cat <<EOF
+Usage: $me: [-<option letter> <options value> | -h]
+Options:
+    -t # Test type, currently support 'local' and 'kube' - Mandatory
+    -b # Test branch - Mandatory
+    -s # Test suite - Mandatory
+    -i # Ignore post cleanup, ignore post test automation cleanup for debug purpose, currently support 'y' and 'n' - Optional
+    -c # Test configuration - currently support 'setup' for install and uninstall PFE container - Optional
+    -h # Display the man page
+EOF
+}
+
+while getopts "t:b:s:i:c:h" OPTION; do
+    case "$OPTION" in
+        t) 
+            TEST_TYPE=$OPTARG
+            # Check if test type argument is corrent
+            if [[ ($TEST_TYPE != "local") && ($TEST_TYPE != "kube") ]]; then
+                echo -e "${RED}Test type argument is not correct. ${RESET}\n"
+                usage
+                exit 1
+            fi
+            ;;
+        b) 
+            TEST_BRANCH=$OPTARG
+            ;;
+        s) 
+            TEST_SUITE=$OPTARG
```

Comment:
What are the options for this argument? We should have a default for this as well and probably list the available options if possible.